pkgs/by-name/kc/kconf/package.nix

```diff
@@ -14,14 +18,25 @@ buildGoModule rec {
   vendorHash = "sha256-REguLiYlcC2Q6ao2oMl92/cznW+E8MO2UGhQKRXZ1vQ=";
 
   ldflags = [
-      "-s" "-w" "-X github.com/particledecay/kconf/build.Version=${version}"
+    "-s"
+    "-w"
+    "-X github.com/particledecay/kconf/build.Version=${version}"
   ];
 
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installShellCompletion --cmd kconf \
+      --bash <($out/bin/kconf completion bash) \
+      --fish <($out/bin/kconf completion fish) \
+      --zsh <($out/bin/kconf completion zsh)
+  '';
+
   meta = with lib; {
     description = "Opinionated command line tool for managing multiple kubeconfigs";
     mainProgram = "kconf";
     homepage = "https://github.com/particledecay/kconf";
     license = licenses.mit;
-    maintainers = with maintainers; [ thmzlt ];
+    maintainers = with maintainers; [thmzlt sailord vinetos];
```

Comment:
```suggestion
    maintainers = with maintainers; [ thmzlt sailord vinetos];
```

Reply:
The space was removed by my formatter 😅 (alejandra) do you really think I should put a space here ?

Reply:
Can you try with `nix run nixpkgs#nixfmt-rfc-style package.nix` ?

Reply:
just did, it should be fine now 👍